cmd/cli/plugin/package/README.md

```diff
@@ -8,46 +8,65 @@ Package and Repository Operations ( Subject to Change ):
 
 ```sh
 >>> tanzu package --help
-Tanzu package management
+Tanzu package management (available, install, installed, repository)
 
 Usage:
+  tanzu package [flags]
   tanzu package [command]
 
 Available Commands:
-    available   Manage available packages
-    install     Install a package
-    installed   Manage installed packages
-    repository  Manage registered package repositories
-
+  available     Manage available packages (get, list)
+  completion    Generate the autocompletion script for the specified shell
+  install       Install package
+  installed     Manage installed packages (create, delete, get, list, update)
+  repository    Manage package repositories (add, delete, get, list, update)
 
 Flags:
-  -h, --help              help for package
-      --log-file string   Log file path
-  -v, --verbose int32     Number for the log level verbosity(0-9)
+      --column strings              Filter to show only given columns
+      --debug                       Include debug output
+  -h, --help                        help for package
+      --kube-api-burst int          Set Kubernetes API client burst limit (default 1000)
```

Comment:
Defaults are super high. What is the reasoning for using these as opposed to defaults set by tools like kubectl or controller-runtime?